### PR TITLE
restructure models

### PR DESCRIPTION
**Pull request by pkuwwt**

### CODE REVIEW

File: app/models/Notice.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+from app import db
+from sqlalchemy import func
+
+class Notice(db.Model):
+    __table_name__ = 'notices'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String, nullable=False)
```
Comment (gaohongwei) [inline app/models/Notice.py:8]: content can be text?

File: app/models/User.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+from app import db
+
+class User(db.Model):
```
Comment (gaohongwei) [inline app/models/User.py:4]: can we add more common fields to user schema?

File: run.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
 from app import app
 
 if __name__ == '__main__':
```
Comment (gaohongwei) [inline run.py:3]: Can update requirement and readme(how to run it in dev/prd env?)

File: app/models/Notice.py (hunk 1 above)
Comment (pkuwwt) [reply app/models/Notice.py:8]: OK

File: app/models/User.py (hunk 2 above)
Comment (pkuwwt) [reply app/models/User.py:4]: Suggestions?

File: run.py (hunk 3 above)
Comment (pkuwwt) [reply run.py:3]: OK. We should also include how to deploy

**Comment by pkuwwt** [comment]
Next, we should determine the following:

  * Deploy method: gunicorn vs. uwsgi
  * Pages, and the storyline for each page

Then I will write the deployment scripts and modules for each page

File: app/models/User.py (hunk 2 above)
Comment (gaohongwei) [reply app/models/User.py:4]: Like password(encrpted), first name, last name, phone etc

**Review by gaohongwei — APPROVED** [review]

**Pull request merged**